starlingbank/__init__.py

```diff
@@ -122,8 +123,43 @@ def get_image(self, filename: str = None) -> None:
             file.write(b64decode(base64_image))
 
 
+"""Representation of a Spending Category."""
+class SpendingCategory:
+
+    def __init__(
```

Comment:
As we have all the data on initialisation, can I suggest the init method also sets the data. For example, perhaps init should have the category, month and year parameters that the existing `update` method has and use those to initialise the data.

Also see my note below about the `update` method.